pkg/service/scheduler/model.go

```diff
@@ -27,6 +27,7 @@ const (
 	UnknownTask        TaskType = "unknown"
 	BackupTask         TaskType = "backup"
 	RestoreTask        TaskType = "restore"
+	One2OneRestoreTask TaskType = "1_1_restore"
```

Comment:
nit: yet another version of writing the 1-1-restore.

Reply:
I thought that using `1-1-restore` when applicable makes sense. In other cases, we can use `one2onerestore`.
@VAveryanov8 why do you think that `1_1_restore` fits better here?

Reply:
This is nit. Please don't spend much time on deciding what name to use. `1_1_restore` is perfectly fine.